include/fiction/algorithms/simulation/sidb/operational_domain.hpp

```diff
@@ -50,108 +138,78 @@ namespace fiction
+    sweep_parameter x_dimension{sweep_parameter::EPSILON_R};
     /**
      * Y dimension sweep parameter.
      */
-    sweep_parameter y_dimension{operational_domain::sweep_parameter::LAMBDA_TF};
-    /**
-     * The parameter point holds parameter values in the x and y dimension.
-     */
-    struct parameter_point
-    {
-        /**
-         * Standard default constructor.
-         */
-        parameter_point() = default;
-        /**
-         * Standard constructor.
-         *
-         * @param x_val X dimension parameter value.
-         * @param y_val Y dimension parameter value.
-         */
-        parameter_point(const double x_val, const double y_val) : x{x_val}, y{y_val} {}
-        /**
-         * X dimension parameter value.
-         */
-        double x;
-        /**
-         * Y dimension parameter value.
-         */
-        double y;
-        /**
-         * Equality operator.
-         *
-         * @param other Other parameter point to compare with.
-         * @return `true` iff the parameter points are equal.
-         */
-        [[nodiscard]] bool operator==(const parameter_point& other) const noexcept
-        {
-            return x == other.x && y == other.y;
-        }
-        /**
-         * Inequality operator.
-         *
-         * @param other Other parameter point to compare with.
-         * @return `true` iff the parameter points are not equal.
-         */
-        [[nodiscard]] bool operator!=(const parameter_point& other) const noexcept
-        {
-            return !(*this == other);
-        }
-        /**
-         * Support for structured bindings.
-         *
-         * @tparam I Index of the parameter value to be returned.
-         * @return The parameter value at the specified index.
-         */
-        template <std::size_t I>
-        auto get() const noexcept
-        {
-            static_assert(I < 2, "Index out of bounds for parameter_point");
-
-            if constexpr (I == 0)
-            {
-                return x;
-            }
-            else  // I == 1
-            {
-                return y;
-            }
-        }
-    };
+    sweep_parameter y_dimension{sweep_parameter::LAMBDA_TF};
     /**
-     * The operational status of the layout for each specified parameter combination. This constitutes the operational
-     * domain. The key of the map is the parameter point, which holds the parameter values in the x and y dimension.
-     * The operational status is stored as the value of the map.
+     * This can store different information depending on the use case. If the operational domain is simulated for
+     * different physical parameters, the parameters are stored with the corresponding operating status.
      */
-    locked_parallel_flat_hash_map<parameter_point, operational_status> operational_values{};
+    locked_parallel_flat_hash_map<Key, Value> operational_domain_values{};
```

Comment:
Why was this renamed again?

Reply:
I thought the new name is better. But I will change it back.